source/contributing.rst

```diff
@@ -29,7 +29,7 @@ How to make an extension
 1. Download the template (https://github.com/pyscf/extension-template), run the
    script in the extension template::
 
-    $ curl https://github.com/pyscf/extension-template/pyscf-extension-template-0.1.0.tar.gz | tar xvzf -
+    $ curl https://github.com/pyscf/extension-template/archive/refs/tags/0.1.0.tar.gz | tar xvzf -
```

Comment:
Neither of these works for me
```
$ curl https://github.com/pyscf/extension-template/archive/refs/tags/0.1.0.tar.gz | tar xvzf -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   141  100   141    0     0    819      0 --:--:-- --:--:-- --:--:--   819

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
$ curl https://github.com/pyscf/extension-template/pyscf-extension-template-0.1.0.tar.gz | tar xvzf -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0    102      0 --:--:-- --:--:-- --:--:--   103

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

Reply:
`curl -L` will follow the redirection done by github

Reply:
@bogdanoff great, that fixes it